ui/scripts/inlineHtml/inlineScripts.js

```diff
@@ -241,7 +241,7 @@ const makeUpdate = function(current, deltas) {
   let EP_delta = modifiersToTotal(deltas['EPS']);
   const current_EP_t = Number(current['EP_t']);
   const EP_t_max = Number(current['EP_t_max']);
-  const new_EP_t = Math.max(0, Math.min(EP_t_max, current_EP_t + EP_delta));
+  const new_EP_t = Math.max(0, Math.min(EP_t_max * 2, current_EP_t + EP_delta));
```

Comment:
This fixes the bug. Pretty straightforward issue.
Note `EP_t_max` is the allocated EP per turn.
In combat, you are allowed to go up to 2x of this value.
This code is calculating usage of EP after an attack.